test/docfx.Test/build/ConfigTest.cs

```diff
@@ -90,10 +90,6 @@ public static void LocConfigConventionTheme(string theme, string locale, string
         [InlineData("https://test.visualstudio.com/_git/abc.bs-Cyrl-BA", "master", "https://test.visualstudio.com/_git/abc", "master", "bs-cyrl-ba")]
         [InlineData("https://github.com/docs.zh-cn", "master-sxs", "https://github.com/docs", "master", "zh-cn")]
         [InlineData("https://github.com/docs.loc", "master-sxs.zh-cn", "https://github.com/docs", "master", "zh-cn")]
-        [InlineData("https://github.com/docs.zh-cn", "test", "https://github.com/docs", "master", "zh-cn")]
-        [InlineData("https://github.com/docs.loc", "test.zh-cn", "https://github.com/docs", "master", "zh-cn")]
-        [InlineData("https://github.com/docs.zh-cn", "test-sxs", "https://github.com/docs", "master", "zh-cn")]
-        [InlineData("https://github.com/docs.loc", "test-sxs.zh-cn", "https://github.com/docs", "master", "zh-cn")]
         public static void LocConfigConventionSourceRepo(string remote, string branch, string expectedSourceRemote, string expectedSourceBranch, string expectedLocale)
```

Comment:
why remove these tests?

Reply:
fallback logic is not covered by TryGetSourceRepo method, it's controlled by restore/build pipeline using different methods

Reply:
we have spec test to cover that case